src/main/java/skype/BasePage.java

```diff
@@ -0,0 +1,154 @@
+package skype;
+
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+import java.util.concurrent.TimeUnit;
+
+import static utils.Timeouts.DEFAULT_WAIT_SECONDS;
+
+
+/**
+ * Created by nastia on 2/26/17.
+ */
+public class BasePage {
+    public String PAGE_URL;
```

Comment:
Should be camelCase for such variables, convention is to use all capital for constants.